crispy_forms/bootstrap.py

```diff
@@ -342,14 +342,20 @@ class Accordion(ContainerHolder):
 
     template = "%s/accordion.html"
 
-    def render(self, form, form_style, context, template_pack=TEMPLATE_PACK, **kwargs):
-        content = ""
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
 
-        # accordion group needs the parent div id to set `data-parent` (I don't
-        # know why). This needs to be a unique id
+        # Accordion needs to have a unique id
         if not self.css_id:
             self.css_id = "-".join(["accordion", str(randint(1000, 9999))])
 
+        # AccordionGroup need to have 'data-parent="#Accordion.id"'
+        for accordion_group in args:
+            accordion_group.data_parent = self.css_id
```

Comment:
In this line is where we could check that `accordion_group` doesn't have an existing `data_parent` and warn if it's the case as they should not be re-used. Is this worth it? How to warn?

At least we should add one line in the documentation saying to not re-use `AccordionGroup`. The warning I'm not sure. Is there any other case in `django-crispy-forms` with a warning? I can't remember now :-)

Reply:
I've had a look at what happens if an `AccordionGroup` is re-used.

Because `AccordionGroup`s would have fields and they would be re-used `django-crispy-forms` shows a logging message (or an exception depending on `CRISPY_FAIL_SILENTLY`): `WARNING:root:A field should only be rendered once: text_input_accordion5`

If we wanted we could warn the user to avoid re-using `AccordionGroup` but probably they will not do or see the error avoid re-using fields. So I think that it might not be necessary? No strong opinion though.